deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -90,6 +90,12 @@ spec:
                 types, but required for libvirt VMs driven by vbmc.
               pattern: '[0-9a-fA-F]{2}(:[0-9a-fA-F]{2}){5}'
               type: string
+            bootMode:
+              description: Select the boot mode of the system. Allowed values are UEFI,
+                legacy. If not set it takes the default boot mode (bios).
```

Comment:
It may not be clear reading this comment that "bios" is same as "legacy". Can we use one of them and not both?

Reply:
It is possible to just ignore the value, by just not adding bootMode into the definition of BareMetalHost CR. What it happens then is that the default behaviour of Ironic is used (legacy)